build-system/tasks/karma.conf.js

```diff
@@ -179,7 +177,10 @@ module.exports = {
     'karma-sauce-launcher',
     'karma-sinon-chai',
     {
-      'middleware:custom': ['factory', function() {return app;}],
+      'middleware:custom': ['factory', function() {
+        var app = require(require.resolve('../server.js'));
```

Comment:
I can't `require(server.js)` here, it will run the `localhost:8000` together with the test server. Shall I mode the code that start a server to a different file? What would be a good solution?